webrtc-stats.html

```diff
@@ -1381,7 +1394,7 @@ <h3>
              double               targetBitrate;
              unsigned long        framesEncoded;
              double               totalEncodeTime;
-             double               averageRTCPInterval;
+             double               averageRtcpInterval;
```

Comment:
Please also add the old spelling to the "obsolete stats" section. It's been burned.

Reply:
was it implemented as yet by anyone? @jan-ivar @aboba and will mention that it was originally  in RTCRTPStreamStats

Reply:
fixed, PTAL @alvestrand

Reply:
We haven't implemented it.